dev/NavigationView/NavigationView.cpp

```diff
@@ -367,6 +368,24 @@ void NavigationView::OnApplyTemplate()
 
     m_accessKeyInvokedRevoker = AccessKeyInvoked(winrt::auto_revoke, { this, &NavigationView::OnAccessKeyInvoked });
 
+    if (SharedHelpers::IsThemeShadowAvailable())
+    {
+#if defined(BUILD_WINDOWS)
+        if (auto splitView = m_rootSplitView.get())
```

Comment:
This should probably be USE_INSIDER_SDK since this will be a public API but is only in the insider SDK